test/test_ruby_parser.rb

```diff
@@ -3766,6 +3766,72 @@ def setup
 
     self.processor = RubyParser::V25.new
   end
+
+  def test_rescue_in_block
```

Comment:
Should these tests be moved to the `TestRubyParserShared25Plus` module? Happy to create a PR if that's the case. Can't see why they wouldn't apply to 2.6+